module3/lessons/Week3/TellingYourStory.md

```diff
@@ -0,0 +1,118 @@
+
+Where might you tell this story?
+* Meet and greet networking events
+* 1:1 coffee chat / informational interview
+* Formal interview
+
+From your professional story, we’ll craft your LinkedIn summary, your resume, and your Terminal portfolio.
+
+Personal branding is the act of telling a consistent story about yourself, building out the details of this story with each profile. As you start this new career, telling your story as a new member of the software industry in as many consistent ways as possible will help you stand out and embrace that identity even more.
+
+## Section 1: Professional Story Telling
+We are all inherent storytellers. Stories are how we connect with each other. From fairy tales to novels to movies to podcasts, stories provide a way for people to share their experiences with others, building empathy and awareness of our universal experiences.
+
+Storytelling is our first step in the job search at Turing. You are a member of the software industry now, and as you start connecting with others in the industry, you need to be able to tell the story of your transition – how did you get here? How do you uniquely make up a part of this industry?
+
+Let’s start by thinking about stories. What do you love about stories? Emma Coats, a former story artist for Pixar, shared her 22 rules for storytelling [here](https://www.washingtonpost.com/blogs/comic-riffs/post/pixar-tips-brave-artist-emma-coats-shares-her-storytelling-wit-and-wisdom-on-twitter%20followher/2012/06/25/gJQADaxd2V_blog.html), and we’ll apply some of them to our storytelling. We’re going to start with rule #10:
+
+*Pull apart the stories you like. What you like in them is a part of you; you’ve got to recognize it before you can use it.*
+
+<section class="call-to-action" markdown="1">
+Think about a story that is meaningful to you and why you connect with it. After time to think on your own, you'll jump into breakout rooms to discuss!
+</section>
+
+## Section 2: Develop Your Professional Story
+
+When you reflected on your meaningful story, you may have thought about what happened to a character in that story, most likely the main character. And that character probably underwent some change throughout the story. That idea of change is going to be a key component of our own professional stories. 
+
+Taking more lessons from Pixar as well as storytelling theory from Joseph Campbell, most stories (in the western tradition at least) follow the 3-Act Arc. Your story should too!
+
+![Acts of a Story](/assets/images/module3/Week3/ActsInAStory.png)
+
+#### Act I (Set-Up)
+
+* Introduction
+* Inciting incident
+* Turning point
+
+#### Act II (Struggle)
+
+* Obstacles & Crisis
+* Looking for ways to solve the problem
+* Learning what it will take to actually solve problem
+
+#### Act III (Finale)
+
+* Climax
+* Denouement/Resolution

```

Comment:
As an aside, let's add the pronunciation of "denouement". In American English, it's day-new-MON, emphasis on the last syllable that rhymes with GONE. (I looked it up, I took Spanish in high school 😄)